La excepción podría levantarse de la siguiente forma:

```ruby 
raise "mensaje de error" 
```